preact/example-customer-account--wishlist--preact/extensions/customer-account-wishlist-extension/src/FullPageExtension.jsx

```diff
@@ -0,0 +1,128 @@
+import '@shopify/ui-extensions/preact';
```

Comment:
Do we want these new extensions to be tsx instead of jsx? just curious if we care about typings being a thing in our tutorials?